demos/segmentation_demo/python/segmentation_demo.py

```diff
@@ -107,7 +107,7 @@ def build_argparser():
     args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
     args.add_argument('-m', '--model', help='Required. Path to an .xml file with a trained model.',
                       required=True, type=Path)
-    args.add_argument('-at', '--architecture_type', help='Required. Specify the model\'s architecture type.',
+    args.add_argument('-at', '--architecture_type', help='Optional. Specify the model\'s architecture type.',
```

Comment:
Could you make it required instead? -at corresponds to -m and it is strange to make assumptions about -at and not about -m.

Reply:
Done